src/Examples/Advanced/CustomProperties/Dictionary_Txy_diagrams.ipynb

```diff
@@ -216,7 +218,15 @@
    "cell_type": "code",
    "execution_count": 4,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2021-05-14 15:33:15 [INFO] idaes.generic_models.properties.core.generic.generic_property: DEPRECATED - definiton of generic property packages is moving to using static classes instead of modules. Please refer to the IDAES documentation.\n"
```

Comment:
We should fix the issue that is causing this warning - it means that the config dictionary for the properties is using a module name to set some options, rather than a class in the module.